roles/k8s_best_practices_certsuite/tasks/pre-run.yml

```diff
@@ -0,0 +1,219 @@
+---
+- name: Verify that kubeconfig is provided
+  ansible.builtin.assert:
+    that:
+      - kbpc_kubeconfig is defined
+      - kbpc_kubeconfig|length
+    fail_msg: "kubeconfig is required to run certsuite"
+
+- name: Fail if certsuite version is lower than v5.1.3
+  when:
+    - kbpc_version is version("v5.1.3", "<")
+    - kbpc_version != "HEAD"
+  ansible.builtin.fail:
+    msg: "k8s_best_practices_certsuite role is no longer compatible with versions lower than v5.1.3"
+
+# Just do this in case preflight test suite is used. This happens if using
+# preflight label. This check is included as a warning for users in case of not
+# defining kbpc_partner_creds properly
+- name: Verify if kbpc_partner_creds is provided when running preflight on certsuite
+  when: "'preflight' in kbpc_test_labels"
```

Comment:
the default is "all" does that include the preflight label? if so you gotta account for both cases

Reply:
I'm not 100% sure that preflight is launched if only providing `all`. For the moment, I'll accept your suggestion and retest; if preflight is not launched if just using `all`, then I'll revert

Reply:
@thekad preflight is not launched if using `all` label, checked here: https://www.distributed-ci.io/jobs/8fdfa989-eccd-4e44-a817-1e7c481b6f8b/tests/6358204a-e7d2-4dd0-91f9-972ac271eefa, so I'll return to the prev code, this is not needed